src/utils/combineReducers.js

```diff
@@ -113,12 +113,16 @@ export default function combineReducers(reducers) {
       throw sanityError;
     }
 
+    var hasChanged = false;
+
```

Comment:
Let's remove this newline; this declaration is directly related to the logic just below.